pinot-query-planner/src/test/resources/queries/WindowFunctionPlans.json

```diff
@@ -3404,7 +3404,7 @@
         "sql": "EXPLAIN PLAN FOR WITH windowfunc AS (SELECT a.col1, ROW_NUMBER() OVER(PARTITION BY a.col2 ORDER BY a.col3) as rownum from a) SELECT a.col1, a.rownum FROM windowfunc AS a where a.rownum < 5",
         "output": [
           "Execution Plan",
-          "\nLogicalProject(col1=[$0], $1=[$3])",
+          "\nLogicalProject(col1=[$0], w0$o0=[$3])",
```

Comment:
@shauryachats : can we also verify that this window query works after this change? (as in it runs successfully)

Reply:
Yes, verified that this window query works correctly in Quickstart.

<img width="1800" alt="Screenshot 2024-10-22 at 3 56 46 PM" src="https://github.com/user-attachments/assets/18b3083f-d1b1-4b1f-ba6d-46b400931503">

<img width="1800" alt="Screenshot 2024-10-22 at 3 56 58 PM" src="https://github.com/user-attachments/assets/a986eb06-b65a-438f-9d0c-548bd4fc6a65">

Reply:
Yes, verified that window queries work in Quickstart.

<img width="1237" alt="Screenshot 2024-10-22 at 3 59 56 PM" src="https://github.com/user-attachments/assets/6c3e060f-f550-4551-a6ee-b73906cfbc29">

<img width="1243" alt="Screenshot 2024-10-22 at 4 00 11 PM" src="https://github.com/user-attachments/assets/1e2d0d36-19bd-4bae-a2be-53f85a6b5aef">